### DISCUSSION TITLE
GitHub education proof submission issue

### DISCUSSION THREAD

**Discussion by birajdotdev** [community/community#70824] [GitHub Education]
### Hi everyone,

I am having an issue with Upload proof of your academic status in GitHub education I have uploaded my letter of admission but whenever I click on the process my application button it redirects me to the same page again and again if I press the button again it says Unable to submit your application. Please try again. What can I do?

**Comment by LFell** [comment]
I have the same issue - have updated bulling info, enabled MFA, logged out, closed browser and then gone back in. Still won't submit, and no way to  get help other than this forum.

**Reply by LFell** [reply]
Just managed to submit my application by uploading the image of my ID card again. Try that and see if it works.
